TATB - 29 : Registration page should contain header

PrabhatJTDeveloper opened the pull request.

Commit:
Technical assistant : Implement header for all pages

PrabhatJTDeveloper committed:
TATB - 29 : Move inline CSS to external SCSS file

charmijtdev commented on src/app/component/header/header.component.scss:
```diff
@@ -34,4 +34,9 @@ button:hover{
 }
```
On line 29 why whole button is applied ? Please add class and please change for span tags too

charmijtdev replied:
on line 4 why container is used directly ? boostrap class ? please remove that too

PrabhatJTDeveloper replied:
I have Remove container

charmijtdev commented on src/app/component/header/header.component.scss:
```diff
@@ -34,4 +34,9 @@ button:hover{
 }
 .modal .modal-header{
```
here why whole modal class is used ? direct from boostrap ?

PrabhatJTDeveloper replied:
I have Remove model

charmijtdev commented on src/app/component/header/header.component.html:
```diff
@@ -97,7 +90,10 @@ <h1>Jyoti Technosoft LLP</h1>
+              <a
+                href="https://jyotitechnosoft.com/"
+                rel="noopener noreferrer"
+                target="_blank"
```
on line 119 Please change this css style by adding align-items-center to class 
  style="align-items: center"
>

charmijtdev replied:
On line 124 Please change inline css to external css 
Same on 143 ,152

PrabhatJTDeveloper replied:
yes I have Remove popup box about and Alert button

charmijtdev commented on src/app/component/header/header.component.html:
```diff
@@ -1,24 +1,17 @@
+    <div class="navigation" *ngIf="userName">
       <div class="container-fluid">
         <div class="collapse navbar-collapse">
           <img src="assets/img/logo1.png" alt="" class="img-responsive" />
```
on line 23 style is used for color please move that into style file

charmijtdev replied:
online 31 please change inline style

PrabhatJTDeveloper replied:
I have Remove 23 line color style

PrabhatJTDeveloper replied:
I have change inline style

PrabhatJTDeveloper committed:
TATB - 30: When click skip button , pop up will remain even after question change
 -Give camelCase to variables
 -Fixed bug when click skip button the popup also swipes on timer end
 -Removed a console error

PrabhatJTDeveloper committed:
TATB:30 :
  -reverted chagnes
  - removed unwated code

PrabhatJTDeveloper committed:
TATB - 29 : Remove inline css and Remove popup box model in header section

PrabhatJTDeveloper committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-29-add-header-section-for-routing-all

PrabhatJTDeveloper committed:
TATB - 29 :  Fixed Merge Conflicts

charmijtdev committed:
Merge branch 'feature/development' into feature/TATB-29-add-header-section-for-routing-all

charmijtdev commented:
Merged to feature/development

PrabhatJTDeveloper committed:
TATB-29-Add header section  in main dashboard and Remove css Add to boostarp class

PrabhatJTDeveloper committed:
Merge branch 'feature/TATB-29-add-header-section-for-routing-all' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-29-add-header-section-for-routing-all

PrabhatJTDeveloper committed:
TATB-29 : Remove unwanted code
